cds/modules/webhooks/tasks.py

```diff
@@ -386,20 +398,24 @@ def progress_updater(seconds, duration):
         head.save(in_output(gif_name), save_all=True,
                   append_images=tail, duration=500)
 
-        def create_object(key, media_type, context_type):
+        def create_object(key, media_type, context_type, timestamp=None):
```

Comment:
maybe we should think about transforming this one to `def create_object(key, **kwargs)`, WDYT?